jenkins_jobs/trigger_host_os_build_from_builds_repo.groovy

```diff
@@ -23,6 +23,7 @@ job('trigger_host_os_build_from_builds_repo') {
       extensions {
```

Comment:
- From commit msg, it seems you'll update all trigger jobs, but you only touch the ones that build packages. Why?
- "include the trigger build URL in their "vote"."  . Not sure how this affects the PR. GitHub creates a link with this URL in the PR web page? If so, mention in commit longer msg to make things clearer.